### PR TITLE
Fixed capitalization typos #9571

### PR DESCRIPTION
**Pull request by ghost**
## Description

These changes cause the structure of the program to be more consistent so the functions have the same code structure.

## Checklist

- [ x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ x] The title of the PR is clear and informative
- [x ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x ] If applicable, the changes made in the PR have proper test coverage
- [ x] For public API changes to cmdlets:
    - [x ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by niander** [comment]
I think this is related to #9571? @pi1024e are these the same changes?

**Comment by pi1024e** [comment]
These are similar changes, but this PR was updated to remove the unnecessary changes because I had to reclone the repo at that point.

File: src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs [hunk 1]
```diff
@@ -33,15 +33,15 @@ namespace Microsoft.Azure.Commands.Profile
     /// <summary>
     /// Cmdlet to log into an environment and download the subscriptions
     /// </summary>
-    [Cmdlet("Connect", AzureRMConstants.AzureRMPrefix + "Account", DefaultParameterSetName = "UserWithSubscriptionId", SupportsShouldProcess=true)]
+    [Cmdlet("Connect", AzureRMConstants.AzureRMPrefix + "Account", DefaultParameterSetName = "UserWithSubscriptionId", SupportsShouldProcess = true)]
```
[the remark this answers is not in the document]
Comment (pi1024e) [reply src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs]: Those were done by the format document function, which I ran to ensure formatting would be proper. I apologize, I will undo this.

File: src/OperationalInsights/OperationalInsights/Models/DataSourcePropertiesPerKind/PSCustomLogDataSourceProperties.cs [hunk 2]
```diff
@@ -252,6 +252,8 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 case JsonToken.String:
                     // Satisfy case in which user uses the old regex property in input
                     return new[] {new RegexDelimiter {MatchIndex = 0, Pattern = (string) reader.Value}};
+                case JsonToken.Null:
```
[the remark this answers is not in the document]
Comment (pi1024e) [reply src/OperationalInsights/OperationalInsights/Models/DataSourcePropertiesPerKind/PSCustomLogDataSourceProperties.cs]: Yes it is!

**Comment by pi1024e** [comment]
Fixed all requested changes!

**Comment by pi1024e** [comment]
Rebased so the changes work with current code.

File: src/DataFactory/DataFactoryV2/help/New-AzDataFactoryEncryptValue.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ PS C:\> New-AzDataFactoryEncryptValue -DataFactoryName "WikiADF" -GatewayName "W
 ```
 
 The first command uses **ConvertTo-SecureString** to convert the specified string to a secure string, and then stores that object in the $Value variable.
-The second command uses **Get-Credential** to collect the windows authentication (user name and password), and then stores that **PSCredential** object in the $Credential variable.
```
Comment (markcowl) [inline src/DataFactory/DataFactoryV2/help/New-AzDataFactoryEncryptValue.md]: Need an update to the DataFactory changelog to mark this module for updating.

File: src/MarketplaceOrdering/MarketplaceOrdering/help/Get-AzMarketplaceTerms.md [hunk 4]
```diff
@@ -38,7 +38,7 @@ RetrieveDatetime  : <RetrieveDatetime>
 ## PARAMETERS
 
 ### -DefaultProfile
-The credentials, account, tenant, and subscription used for communication with azure.
+The credentials, account, tenant, and subscription used for communication with Azure.
```
Comment (markcowl) [inline src/MarketplaceOrdering/MarketplaceOrdering/help/Get-AzMarketplaceTerms.md]: Need an update to the MarketPlaceOrdering changelog to mark this module for update

**Comment by markcowl** [comment]
@pi1024e Need changelog entried for the other affected modules.  Note - thsi is a versioning trigger fo us, we can't version the moduels without a changelog update.

**Comment by pi1024e** [comment]
Changes have been made. Please tell me if this is not sufficient enough or there needs to be any changes.

**Comment by pi1024e** [comment]
@markcowl I made the appropriate changes. Please let me know if there should be other changes. Thank you!

**Pull request merged**